apps/api/src/view-models/edgeConnectionViewModels/note.view-model.ts

```diff
@@ -7,7 +8,7 @@ import {
 import cloneToPlainObject from '../../lib/utilities/cloneToPlainObject';
 import { BaseViewModel } from '../buildViewModelForResource/viewModels/base.view-model';
 
-export class NoteViewModel extends BaseViewModel {
+export class NoteViewModel extends BaseViewModel implements INoteViewModel {
```

Comment:
The query service should be typed to return the interface, not the concrete (backend) view model instance.